extensions/wikia/LanguageWikisIndex/LanguageWikisIndexHooks.class.php

```diff
@@ -65,6 +65,40 @@ public static function onGetHTMLBeforeWikiaPage( &$beforeWikiaPageHtml ) {
 		return true;
 	}
 
+	public static function onMercuryWikiVariables( array &$wikiVariables ): bool {
+		if (
+			self::isEmptyDomainWithLanguageWikis() ||
+			self::isClosedWikiWithLanguageWikis()
```

Comment:
This duplicates the `WikiFactory::isLanguageWikisIndex` method you're adding it this PR.